roles/etcd/tasks/install.yml

```diff
@@ -15,3 +15,9 @@
 
 - name: Create etcd2 binary symlink
   file: src=/usr/local/bin/etcd dest=/usr/local/bin/etcd2 state=link
+
+- name: install required python module 'httplib2'
+  apt:
+    name: "python-httplib2"
+    state: present
+  when: inventory_hostname == groups['kube-master'][0] or inventory_hostname == groups['etcd'][0]
```

Comment:
This is not directly related to the etcd role, you're right.
These dependencies should be part of a "common" role, this work will be done in the same time with #34